R/pkgs/report_generation/inst/rmarkdown/templates/county_report/skeleton/skeleton.Rmd

```diff
@@ -0,0 +1,279 @@
+---
```

Comment:
I pushed some formatting changes to dataseed_merge, removed defaults (e.g. included_state: "MD") that need to be input for each report